tests/endpoint-test.js

```diff
@@ -1293,7 +1293,69 @@ describe('Endpoint', function() {
 				});
 			});
 		});
+	});
+
+
+	describe('Collaboration Whitelists', function() {
+		describe('create()', function() {
+			it('should make a post request to create a domain collaboration whitelisting', function() {
+				var post_fixture = getFixture('collaboration-whitelists/post_collaboration_whitelists_200'),
+					domain = 'test15.com',
+					direction = 'both',
+					expected_post_body = {
+						domain: 'test15.com',
+						direction: 'both'
+					};
+
+				apiMock.post('/2.0/collaboration_whitelist_entries', expected_post_body)
+					.matchHeader('Authorization', function(authHeader) {
+						assert.equal(authHeader, 'Bearer ' + TEST_ACCESS_TOKEN);
+						return true;
+					})
+					.matchHeader('User-Agent', function(uaHeader) {
+						assert.include(uaHeader, 'Box Node.js SDK v');
+						return true;
+					})
+					.reply(200, post_fixture);
+
+				return basicClient.collaborationWhitelists.create(domain, direction)
+					.then(collabWhitelist => {
+						assert.deepEqual(collabWhitelist, JSON.parse(post_fixture));
+					});
+			});
+		});
+
+		describe('addUser()', function() {
```

Comment:
These names will need to be updated with the changes in the actual implementation